src/workerd/api/actor.c++

```diff
@@ -133,6 +137,21 @@ jsg::Ref<DurableObject> DurableObjectNamespace::get(
     jsg::Ref<DurableObjectId> id,
     jsg::Optional<GetDurableObjectOptions> options,
     CompatibilityFlags::Reader featureFlags) {
+  return getImpl(ActorGetMode::GET_OR_CREATE, kj::mv(id), kj::mv(options), kj::mv(featureFlags));
+}
+
+jsg::Ref<DurableObject> DurableObjectNamespace::getExisting(
+    jsg::Ref<DurableObjectId> id,
+    jsg::Optional<GetDurableObjectOptions> options,
+    CompatibilityFlags::Reader featureFlags) {
+  return getImpl(ActorGetMode::GET_EXISTING, kj::mv(id), kj::mv(options), kj::mv(featureFlags));
```

Comment:
If I'm reading this correctly, the `KJ_ASSERT` in `server.c++` could cause this to fail with an internal error rather than a useful user-facing error when called on workerd, right? If that's the case, the `KJ_ASSERT` should likely be changed to a `JSG_REQUIRE`? Or, perhaps an API on `IoContext` that asks if `GET_EXISTING` is supported and throws an appropriate JavaScript error here?

Reply:
Yep, agreed.